Dockerfile

```diff
@@ -49,7 +49,8 @@ RUN dpkg --add-architecture i386 && \
         libasound2-dev \
         libc6:i386 \
         libstdc++6:i386 \
-        libsdl1.2-dev
+        libsdl1.2-dev \
+        xutils-dev
```

Comment:
I don't rememeber anymore why I added this. I'll update this coment when I have figured that out.

Reply:
ok found it. I need this for `lndir`